edb/server/main.py

```diff
@@ -233,7 +233,7 @@ async def _run_server(
 
 
 def _generate_cert(
-    cert_dir: pathlib.Path, listen_host: str
+    cert_dir: pathlib.Path, listen_host: Union[str, Iterable]
```

Comment:
Maybe fix the caller code to always pass a `list[str]`?

Reply:
Sure thing! I would also like to update `edgedb-server` to take multiple `-I` values, e.g. `edgedb-server -I 127.0.0.1 -I 192.168.2.22`, because one could anyway set the system config `listen_addresses` to multiple hosts.